### ISSUE TITLE
[Feature] Associated performers and studios on performer page

### ISSUE THREAD

**Issue by QxxxGit** [stashapp/stash#1596]
I think it would be neat to have associated performers (however the user wants to define this) and studios below the details on the Performer page. It would make the details page feel more life-like, and it would make it easier to jump to related performers, and the studios they have been with.

**Describe the solution you'd like**
User goes to a performer's edit page. There is an input field for Associated Performers that allows you to lookup performers. You can add as many as you like. On save, you now will view the details tab on the performer page. Below the details are 2 new sections: Associated Performers and Studios.

The Associated Performers section would include the associated performer's picture (and maybe name), while the Studios section would include the logos. You can get an idea of how the Associated Performer's section would look by going here and scrolling down to "Related Films" https://letterboxd.com/film/batman-1989/

The only database change I think would be needed is to link-up the performers. Studios have a performers tab so that link is there already.

I don't think the Associated Performers section should use the entire performer card, but that can be up for discussion. I prefer the look of the link I provided. It's simple and clean.

**Comment by squanchie** [comment]
Yes! Was just about to request this "recommended/associated" section myself. It would go a long way in connecting like performers, whether it's their appearance, performances, etc.

**Comment by echo6ix** [comment]
Displaying a sub-section of related metadata is a neat idea.

I think the deal killer could be having to manually designate relationships/associations. It would be much better if Stash did this automatically using similarities in `performer tags`, `birthday`, `height`, and other metadata. 

Also I am not sure I am not fully understanding what you mean by association. On other sites/apps sometimes this refers to relationships with other performers based on shared performing in scenes together.

Either way, both are neat concepts.

**Comment by QxxxGit** [comment]
> 
> 
> Displaying a sub-section of related metadata is a neat idea.
> 
> I think the deal killer could be having to manually designate relationships/associations. It would be much better if Stash did this automatically using similarities in `performer tags`, `birthday`, `height`, and other metadata.
> 
> Also I am not sure I am fully understanding what you mean by association. On other sites/apps sometimes this refers to relationships with other performers based on performances in scenes together.
> 
> Either way, both are neat concepts.

The reason I suggested manually linking performers is because some performers work together often, but don't fall under a studio (i.e. Xev B and Leia). These two might have very different performer tags; so if Stash did this automatically, they most likely wouldn't be in the associated performers area because their characteristics are very different.

That being said, I do agree that an automated metadata association would be neat.

**Comment by QxxxGit** [comment]
#3563 satisfies my request so I'm closing this longstanding issue.
